libs/aws/tests/unit_tests/chat_models/test_bedrock_converse.py

```diff
@@ -87,7 +87,14 @@ def test__messages_to_bedrock() -> None:
     messages = [
         SystemMessage(content="sys1"),
         SystemMessage(content=["sys2"]),
-        SystemMessage(content=[{"text": "sys3"}, {"type": "text", "text": "sys4"}]),
+        SystemMessage(
+            content=[
+                {"text": "sys3"},
+                {"type": "text", "text": "sys4"},
+                {"guardContent": {"text": {"text": "sys5"}}},
+                {"type": "guard_content", "text": "sys6"},
```

Comment:
The first form is understood, and matches with Bedrock guardrails documentation. Are we introducing the second form with `{"type": "guard_content"}`? Referring to this doc from the converse API.
https://docs.aws.amazon.com/bedrock/latest/userguide/guardrails-use-converse-api.html

Reply:
yea we are introducing {"type": "guard_content", ...}. this is meant to imitate the anthropic format, since we support anthropic style blocks otherwise. but since this isn't a type actually supported by anthropic id be ok dropping it if we don't like it